tests/integration/conftest.py

```diff
@@ -444,10 +445,76 @@ def azure_cli_authenticated(self):
     def azure_subscription_id(self):
         return self._env_or_skip("TEST_SUBSCRIPTION_ID")
 
+    @cached_property
+    def config(self) -> WorkspaceConfig:
+        return WorkspaceConfig(
+            warehouse_id=self._env_or_skip("TEST_DEFAULT_WAREHOUSE_ID"),
+            inventory_database=self.inventory_database,
+            connect=self.workspace_client.config,
+            renamed_group_prefix=f'tmp-{self.inventory_database}-',
+        )
+
+    @cached_property
+    def installation(self):
+        return MockInstallation()
```

Comment:
Are you sure you want to use in-memory installation for something that calls a job?..